### PR TITLE
Document the tags used on the games

### PR DESCRIPTION
**Pull request by GalaxyGamingBoy**
With the results of discussion on issue #1454, the most common opinion was to educate the users on using and filling out the tags rather than abstracting them entirely making them confused more in the long run. This PR documents the process that a user would need to follow in order to fill in the tags and can be found in the sprig submit instructions (/get path)

### CODE REVIEW

File: src/pages/get.astro [hunk 1]
```diff
@@ -1,68 +1,184 @@
...
-						<h2>Download your game file</h2>
-						<p>In the top left Sprig menu, press "Download". Your file should start downloading immediately.</p>
+						<h2>Fill in the game metadata</h2>
+						<p>
+							In the header of your game you should see a section
+							about game metadata. In order to publish your game
+							to the sprig gallery website you will need to fill
+							it in according to the following instructions. For
+							any assistance feel free to sent a message on the
+							<a href="https://hackclub.com/slack/" target="_blank">#sprig</a> channel on the <a href="https://hackclub.com/slack/" target="_blank">slack</a>
```
Comment (JosiasAurel) [inline src/pages/get.astro:55]: replace the link for `#sprig` with `https://hackclub.slack.com/archives/C02UN35M7LG`

File: src/pages/get.astro [hunk 2]
```diff
@@ -1,68 +1,184 @@
...
 						<p class="step-number">Step 0:</p>
-						<h2>Download your game file</h2>
-						<p>In the top left Sprig menu, press "Download". Your file should start downloading immediately.</p>
+						<h2>Fill in the game metadata</h2>
+						<p>
+							In the header of your game you should see a section
+							about game metadata. In order to publish your game
+							to the sprig gallery website you will need to fill
+							it in according to the following instructions. For
+							any assistance feel free to sent a message on the
```
Comment (JosiasAurel) [inline src/pages/get.astro:54]: should be `send` and not `sent`

File: src/pages/get.astro [hunk 3]
```diff
@@ -1,68 +1,184 @@
...
+						<p class="step-number">Step 0B:</p>
+						<h2>Adding tags</h2>
+						<p>
+							Tags on sprig are an easy way for players to
+							categorize their games based on their content. For
+							example a game that has a tag adventure will have
+							adventurous gameplay. You can see all of the tags
+							that are curently used for reference on the <a
+								href="/gallery">Sprig Gallery</a
+							> <strong>tag selection dropbox</strong> but you can always add new ones. The metadata tag @tags
```
Comment (JosiasAurel) [inline src/pages/get.astro:98]: ~dropbox~ dropdown

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
there are a few corrections to be made. 
also please avoid formatting the code in the future it makes it hard to see the code diff

File: src/pages/get.astro (hunk 3 above)
Comment (GalaxyGamingBoy) [reply src/pages/get.astro:98]: my bad

**Comment by leomcelroy** [comment]
What sort of feedback are you looking for on this review? I can describe what it's supposed to do but I think reviewing the code itself for correctness is outside my purview.

**Comment by JosiasAurel** [comment]
> What sort of feedback are you looking for on this review? I can describe what it's supposed to do but I think reviewing the code itself for correctness is outside my purview.

I am looking for feedback on the submission steps - [see updated steps here](https://sprig-1qpblsv1q-gamer.vercel.app/get)

**Review by grymmy — APPROVED** [review]
Seems fine to me, if verbose (could be a smaller singular section).

**Comment by leomcelroy** [comment]
Only have title and author in the the default file. The additional information should be referenced in the submission doc not the default example.

**Comment by GalaxyGamingBoy** [comment]
@leomcelroy Removed the other tags in the template

**Comment by GalaxyGamingBoy** [comment]
Before:
image

After:
image

**Review by grymmy — CHANGES_REQUESTED** [review]
Plz make more concise and I believe this section is best left to end

**Comment by GalaxyGamingBoy** [comment]
> Plz make more concise and I believe this section is best left to end

Okay, I can make it more concise. But the position should ideally be first, since you kind of need it before committing the game/pr

**Comment by GalaxyGamingBoy** [comment]
image

**Review by JosiasAurel — APPROVED** [review]

**Pull request closed without merging**